### PR TITLE
fix: Use smithy-swift package manifest in root of project

### PR DESCRIPTION
**Pull request by jbelkins**
## Description of changes
https://github.com/awslabs/smithy-swift/pull/437 deletes a duplicate `Package.swift` file that was contained in a subfolder of the main project.  This project was making reference to the now-deleted `Package.swift` to import the `ClientRuntime` library in the following places:
- `Package.swift` located in the project root directory
- `Package.swift` located in the `AWSClientRuntime/` subfolder
- `Package.swift` located in the `codegen/` subfolder

This project now imports `ClientRuntime` using only the `Package.swift` file in the root of the `smithy-swift` project.

## New/existing dependencies impact assessment, if applicable
No new dependencies were added to this change, but one dependency (`smithy-swift`) is imported differently.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ganeshnj** [comment]
Check https://github.com/awslabs/smithy-swift/pull/437#issuecomment-1252952009

**Comment by jbelkins** [comment]
@ganeshnj Turns out there were two additional `Package.swift` files in this project whose references to `ClientRuntime` had to be adjusted.  Those changes are made in this PR and in branch `jbe/swift_package_reorg` (so that https://github.com/awslabs/smithy-swift/pull/437 builds successfully.)

File: AWSClientRuntime/Package.swift [hunk 1]
```diff
@@ -42,11 +42,11 @@ if let smithySwiftDir = ProcessInfo.processInfo.environment["SMITHY_SWIFT_CI_DIR
    let crtDir = ProcessInfo.processInfo.environment["AWS_CRT_SWIFT_CI_DIR"] {
     package.dependencies += [
         .package(name: "AwsCrt", path: "\(crtDir)"),
-        .package(name: "ClientRuntime", path: "\(smithySwiftDir)/Packages")
+        .package(name: "ClientRuntime", path: "\(smithySwiftDir)")
```
Comment (ganeshnj) [inline AWSClientRuntime/Package.swift:45]: nit: string interpolation is not needed at most `path` params.
```suggestion
        .package(name: "ClientRuntime", path: smithySwiftDir)
```

**Review by ganeshnj — APPROVED** [review]

File: AWSClientRuntime/Package.swift (hunk 1 above)
Comment (jbelkins) [reply AWSClientRuntime/Package.swift]: Fixed this and several other places where interpolation is unneeded

**Pull request merged**